### PR TITLE
docker-ckan based workflow for ckanext-datajson local development

### PR DESCRIPTION
**Pull request by thejuliekramer**
docker-ckan based workflow for ckanext-datajson local development

https://github.com/GSA/datagov-ckan-multi/issues/208

### CODE REVIEW

**Comment by thejuliekramer** [comment]
The build is failing because the datagov main branch CI is failing. I believe @avdata99 fixes that here GSA/ckanext-datajson#54. Is it possible to merge this before mine so we see the clean build?

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ ckanext-datajson
 [![Build Status](https://travis-ci.org/GSA/ckanext-datajson.svg?branch=master)](https://travis-ci.org/GSA/ckanext-datajson)
 [![Codacy Badge](https://api.codacy.com/project/badge/c4816041270448b6a0c9760933065b7e)](https://www.codacy.com/app/alexandr-perfilov/ckanext-datajson)
 [![Coverage Status](https://coveralls.io/repos/GSA/ckanext-datajson/badge.svg?branch=master&service=github)](https://coveralls.io/github/GSA/ckanext-datajson?branch=master)
+[![CircleCI](https://circleci.com/gh/GSA/ckanext-datajson.svg?style=svg)](https://circleci.com/gh/GSA/ckanext-datajson)
```
Comment (adborden) [inline README.md]: Would you delete the other badges? They are no longer valid.

File: README.md [hunk 2]
```diff
@@ -19,6 +20,8 @@ at http://ckanhostname/pod/validate.
 Installation
 ------------
 
+To install using a docker development environment, refer [here](ckan-docker/README.md)
```
Comment (adborden) [inline README.md]: Let's pull this inline instead of having it in a separate document. The docker workflow is the primary way to setup a development environment.

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+version: "3.2"
+services:
+  ckanextdatajson:
```
Comment (adborden) [inline docker-compose.yml]: maybe we'll just call this `ckan` for simplicity.

File: ckanext/datajson/tests/mock_datajson_source.py [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 import SocketServer
 from threading import Thread
-import logging
+
+from ckanext import datajson
+
 log = logging.getLogger("harvester")
 
 PORT = 8998
+SAMPLES_PATH = os.path.join(datajson.__path__[0], 'tests/datajson-samples')
```
Comment (adborden) [inline ckanext/datajson/tests/mock_datajson_source.py]: Same comment as https://github.com/GSA/ckanext-geodatagov/pull/123#discussion_r375941117, there should be a better way to do this so it doesn't break in all these corner cases.

Comment (thejuliekramer) [reply ckanext/datajson/tests/mock_datajson_source.py]: I can't use the file method here because datajson is being declared as a namespace package in the init file
